### PR TITLE
KasurajaAlgorithm C++ implementation

### PR DESCRIPTION
**Pull request by BhanujaAggarwal**
Fixes #

#### Checklist:
- [ ] 4 space indentation.
- [ ] Coding conventions are followed.
- [ ] Input is taken dynamically.
- [ ] Sample Input / Output is added at the end of file.
- [ ] Logic Documentation (Comments).
- [ ] File names are correct.

Make sure these boxes are checked before your pull request (PR) is ready to be reviewed and merged. Thanks!

#### Changes proposed in this pull request:
- 
- 

#### Languages Used:
- 
- 

#### Files Added:
- 
- 

> We're happy to help you get this ready -- don't be afraid to ask for help.

Thanks!

### CODE REVIEW

**Comment by somya-kapoor** [comment]
Please mention the issue number you are trying to fix.
Also, complete the PR checklist properly.

File: KasurajaAlgorithm.cpp [hunk 1]
```diff
@@ -0,0 +1,116 @@
+#include<bits/stdc++.h>
```
Comment (jainaman224) [inline KasurajaAlgorithm.cpp]: Add this file to a folder and fix file name. Have a look at existing files.

File: KasurajaAlgorithm.cpp [hunk 2]
```diff
@@ -0,0 +1,116 @@
...
+using namespace std; 
+
+class Graph 
+{ 
+	int V;
```
Comment (jainaman224) [inline KasurajaAlgorithm.cpp]: Indent using 4 spaces.

File: KasurajaAlgorithm.cpp [hunk 3]
```diff
@@ -0,0 +1,116 @@
...
+		{ 
+			gr.DFSUtil(v, visited); 
+			cout << endl; 
+		} 
+	} 
+} 
+
+int main() 
+{ 
+	Graph g(8);
```
Comment (jainaman224) [inline KasurajaAlgorithm.cpp]: Take input from stdin.

File: KasurajaAlgorithm.cpp [hunk 4]
```diff
@@ -0,0 +1,116 @@
...
+	g.addEdge(0,4);
+	g.addEdge(4,2);
+	g.addEdge(3, 4);
+	g.addEdge(0, 3); 
+
+	cout<<"By Kasuraja Algorithm the strongly connected components in given graph are"<<endl;
+	g.KasurajaAlgorithm(); 
+
+	return 0; 
+}
```
Comment (jainaman224) [inline KasurajaAlgorithm.cpp]: Add sample input and output.

File: KasurajaAlgorithm.cpp [hunk 5]
```diff
@@ -0,0 +1,116 @@
...
+
+	list<int>::iterator i; 
+	for(i = adj[v].begin(); i != adj[v].end(); ++i) 
+		if(!visited[*i]) 
+			fillOrder(*i, visited, Stack); 
+
+	Stack.push(v); 
+} 
+
+void Graph::KasurajaAlgorithm()
```
Comment (jainaman224) [inline KasurajaAlgorithm.cpp]: Add some comments throughout this file to help reader understand.

**Comment by somya-kapoor** [comment]
@BhanujaAggarwal  Are you working on it?

**Comment by BhanujaAggarwal** [comment]
@somya-kapoor yes I'm working on it

**Comment by jainaman224** [comment]
@BhanujaAggarwal Are you working on it?

**Comment by BhanujaAggarwal** [comment]
@jainaman224 Yes, I  have fixed the mistakes.

**Comment by somya-kapoor** [comment]
@BhanujaAggarwal Please delete the existing PR if you are creating a new PR for the same issue.

**Pull request closed without merging**